src/app/page/home/home.component.ts

```diff
@@ -21,6 +21,8 @@ export class HomeComponent implements OnInit {
   }
 
   arreglo(data: any[]): void {
-    this.productsFiltros = this.product.filter(item => data.includes(item.filtro))
+    this.productsFiltros = this.product.filter(item => data.includes(item.filtro) || data[0] == item.nombre.toLowerCase())
```

Comment:
Talvez deberia crear dos metodos de filtro, por tipo y por nombre, pero si quiere mantenerlo y mejora el buscador seria asi
```suggestion
    this.productsFiltros = this.product.filter(item => data.includes(item.filtro) || data.toLowerCase().includes(item.nombre.toLowerCase()))
```